### DISCUSSION TITLE
什么时候支持在webStrom中的生成提交消息,并像vscode中支持设置回答默认语言为简体中文

### DISCUSSION THREAD

**Discussion by yuhanaaa** [community/community#116715] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

什么时候支持在webStrom中的生成提交消息,并像vscode中支持设置回答默认语言为简体中文

**Comment by Akash1134** [comment]
Hi @yuhanaaa , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
